runconfig/parse.go

```diff
@@ -65,6 +65,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 		flCpuShares       = cmd.Int64([]string{"c", "-cpu-shares"}, 0, "CPU shares (relative weight)")
 		flCpusetCpus      = cmd.String([]string{"#-cpuset", "-cpuset-cpus"}, "", "CPUs in which to allow execution (0-3, 0,1)")
 		flCpusetMems      = cmd.String([]string{"-cpuset-mems"}, "", "MEMs in which to allow execution (0-3, 0,1)")
+		flBlkioWeight     = cmd.Int64([]string{"-blkio-weight"}, 0, "Block IO relative weight, range from 10 to 1000")
```

Comment:
Yes, that's default of the kernel, not set by Docker.
But `nil` usually for struct or pointer, technically we call it default initial value, and default initial value for int64 is 0, you can consider it not a magic number :)